src/services/jsDoc.ts

```diff
@@ -1,48 +1,92 @@
 /* @internal */
 namespace ts.JsDoc {
     const jsDocTagNames = [
+        "abstract",
+        "access",
+        "alias",
+        "arg",
+        "argument",
+        "async",
         "augments",
         "author",
-        "argument",
         "borrows",
         "callback",
         "class",
+        "classDesc",
+        "const",
         "constant",
         "constructor",
         "constructs",
+        "copyright",
         "default",
         "deprecated",
         "description",
+        "emits",
+        "enum",
         "event",
         "example",
+        "exports",
         "extends",
+        "external",
         "field",
+        "file",
         "fileOverview",
+        "fires",
+        "func",
         "function",
+        "generator",
+        "global",
+        "hideConstructor",
+        "host",
         "ignore",
+        "implements",
         "inheritDoc",
         "inner",
+        "instance",
+        "interface",
+        "kind",
         "lends",
+        "license",
         "link",
+        "linkCode",
+        "linkPlain",

```

Comment:
i do not see any of these in http://usejsdoc.org/

Reply:
`link`, `linkCode` and `linkPlain` are defined in the "Inline Tags" tags section on the page http://usejsdoc.org/#inline-tags

More about the difference between block tags and inline tags can be found here: http://usejsdoc.org/about-block-inline-tags.html